### PR TITLE
Inserted Example 2

### PR DESCRIPTION
**Pull request by Skatterbrainz**
Inserted Example 2 to show using -TitleIncludes parameter usage

### CODE REVIEW

**Comment by Skatterbrainz** [comment]
Oops - I pasted the additional lines in the wrong example. The "Windows 10" lines should be under the new Example 2.

File: docset/winserver2022-ps/updateservices/Get-WsusProduct.md [hunk 1]
```diff
@@ -42,6 +42,19 @@ This command gets all products defined by the WSUS server.
...
+```text
+PS C:\> Get-WsusProduct -TitleIncludes "Windows 10"
+Title                                                     ID                                  
+-----                                                     --                                  
+Windows 10 and later drivers                              05eebf61-148b-43cf-80da-1c99ab0b8699
+Windows 10 and later Dynamic Update                       e4b04398-adbd-4b69-93b9-477322331cd3
+Windows 10 and later GDR-DU                               abc45868-0c9c-4bc0-a36d-03d54113baf4
+```
+
+This command gets all products where Title includes Windows 10
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/updateservices/Get-WsusProduct.md:54]:
```suggestion
This command gets all products where the product title includes Windows 10.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Skatterbrainz, kindly commit the suggested change. Many thanks.

**Comment by scanum** [comment]
We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Pull request closed without merging**